### PR TITLE
[14.0][MIG] apps_product_creator: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT29952

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

File: apps_product_creator/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+#~ msgid "Icon Image"
+#~ msgstr "Icono"
+
+#~ msgid "odoo.module"
+#~ msgstr "odoo.module"
+
+#, fuzzy
+#~ msgid "odoo.module.version"
+#~ msgstr "odoo.module"
```
Comment (victoralmau) [inline apps_product_creator/i18n/es.po]:
```suggestion
```

File: apps_product_creator/i18n/it.po [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+msgid "Website (Manifest)"
+msgstr "Sito web (manifest)"
+
+#. module: apps_product_creator
+#: model:product.public.category,name:apps_product_creator.ecommerce
+msgid "eCommerce"
+msgstr "E-commerce"
+
+#~ msgid "Icon Image"
+#~ msgstr "Immagine icona"
```
Comment (victoralmau) [inline apps_product_creator/i18n/it.po]:
```suggestion
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
@victoralmau changes done

**Review by victoralmau — APPROVED** [review]

File: apps_product_creator/models/product_product.py [hunk 3]
```diff
@@ -81,7 +81,7 @@ class ProductProduct(models.Model):
         store=True,
     )
     app_development_status = fields.Selection(
-        "Module maturity",
+        name="Module maturity",
```
Comment (pedrobaeza) [inline apps_product_creator/models/product_product.py]:
```suggestion
        string="Module maturity",
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**